src/components/SubscriptAvatar.js

```diff
@@ -63,23 +63,21 @@ const SubscriptAvatar = (props) => {
                     type={props.mainAvatar.type}
                 />
             </Tooltip>
-            <View
-                style={[
-                    props.size === CONST.AVATAR_SIZE.SMALL ? styles.secondAvatarSubscriptCompact : styles.secondAvatarSubscript,
-                    StyleUtils.getBackgroundAndBorderStyle(props.backgroundColor),
-                    StyleUtils.getAvatarBorderStyle(props.size, props.secondaryAvatar.type),
-                ]}
-            >
-                <Tooltip text={props.secondaryTooltip}>
-                    <Avatar
-                        source={props.secondaryAvatar.source}
-                        size={props.size === CONST.AVATAR_SIZE.SMALL ? CONST.AVATAR_SIZE.SMALL_SUBSCRIPT : CONST.AVATAR_SIZE.SUBSCRIPT}
-                        fill={themeColors.iconSuccessFill}
-                        name={props.secondaryAvatar.name}
-                        type={props.secondaryAvatar.type}
-                    />
-                </Tooltip>
-            </View>
+            <Tooltip text={props.secondaryTooltip}>
+                <Avatar
+                    imageStyles={null}
```

Comment:
Hi this PR caused https://github.com/Expensify/App/issues/19905

More details - https://github.com/Expensify/App/issues/19905#issuecomment-1593690817

Reply:
Thanks @rushatgabhane We didn't test with a real image indeed. Nice catch!